src/lib.rs

```diff
@@ -50,6 +50,7 @@
 #![doc(test(attr(allow(unused_variables), deny(warnings))))]
 #![no_std]
 #![cfg_attr(feature = "simd_support", feature(portable_simd))]
+#![allow(unexpected_cfgs)]
```

Comment:
This can be removed after renaming `doc_cfg` to `docsrs`.

Reply:
I probably will leave fixing it for a later PR. We also should consider using `doc_auto_cfg` (it has certain issues, but I am not sure they affect us).

Reply:
Can we leave it to a later PR anyway?